assets/base.css

```diff
@@ -1584,6 +1578,8 @@ input[type='checkbox'] {
   align-items: center;
   justify-content: center;
   color: rgb(var(--color-foreground));
+  margin: 0;
```

Comment:
Do we need this? I looks like button is set to `0em` by default.

Reply:
I removed it